src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -236,7 +236,7 @@ private CookieOptions BuildCookieOptions()
                 if (signInContext.Properties.IsPersistent)
                 {
                     var expiresUtc = signInContext.Properties.ExpiresUtc ?? issuedUtc.Add(Options.ExpireTimeSpan);
-                    signInContext.CookieOptions.Expires = expiresUtc.ToUniversalTime().DateTime;
+                    signInContext.CookieOptions.Expires = expiresUtc.LocalDateTime;
```

Comment:
@brentschmaltz  I think using LocalDateTime is the simplest workaround to make the conversion work as expected. Maybe DateTimeOffset class should have a `UniversalDateTime` property which set its `Kind` value to `DateTimeKind.Utc`.